src/background.xml

```diff
@@ -5,66 +5,76 @@
       Transport Capacity measurement was known to be well beyond
       our capabilities. Even at the time that Framework for IP
       Performance Metrics 
-      <xref target="RFC3148" /> was written we knew that we didn't
```

Comment:
Then the title of 2330, "Framework for IP Performance Metrics", shouldn't appear in the text. Maybe "Empirical BTC Metrics" instead.